build/grafana/dashboard-apiserver-requests.yaml

```diff
@@ -74,27 +85,31 @@ data:
           "linewidth": 1,
           "links": [],
           "nullPointMode": "null",
+          "options": {
+            "alertThreshold": true
+          },
           "paceLength": 10,
           "percentage": false,
+          "pluginVersion": "9.3.6",
```

Comment:
Just for my learning purpose, why the `pluginVersion` is not explicitly set previously?

Reply:
Like I said, this was made by "save" in the UI then copying the JSON blob. I have no idea why some of these values changed - I suspect at some point they extended the schema to include the plugin version so you could "bottle" charts like this.